content/plotly/concepts/graph-objects/terms/scatterpolar/scatterpolar.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.Scatterpolar()'
+Description: 'Creates a scatter plot on polar axes.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Plotly'
+  - 'Graphs'
+  - 'Data'
+  - 'Values'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.Scatterpolar()`** function in Plotly is used to create scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes that are often used to represent periodic data. 
```

Comment:
```suggestion
The **`.Scatterpolar()`** function in Plotly creates scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes often used to represent periodic data. 
```
Edited the sentence for more clarity and understanding.

Reply:
I personally love this clarification. Thanks much.